### PR TITLE
Vcpkg fix

### PR DESCRIPTION
**Pull request by sean-mcmanus**

### CODE REVIEW

File: Extension/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 # C/C++ for Visual Studio Code Change Log
 
+## Version 0.17.7: July 16, 2018
+* Add `${vcpkgRoot}` variable. [#1817](https://github.com/Microsoft/vscode-cpptools/issues/1817)
+* Skip automatic parsing of source files in Mac system framework paths. [#2156](https://github.com/Microsoft/vscode-cpptools/issues/2156)
+* Fix indexing of the entire root drive on Windows when no is folder open. [#2216](https://github.com/Microsoft/vscode-cpptools/issues/2216)
+* Fix infinite CPU/memory usage until it crashes with `#include` code actions when no folder is open. [#2225](https://github.com/Microsoft/vscode-cpptools/issues/2225)
```
Comment (bobbrow) [inline Extension/CHANGELOG.md]: TMI - can you just say "infinite loop"?

File: Extension/src/LanguageServer/client.ts [hunk 2]
```diff
@@ -322,6 +323,15 @@ class DefaultClient implements Client {
             storagePath = path.join(storagePath, serverName);
         }
 
+         // Check for vcpkg instance.
+        if (existsSync(util.getVcpkgPathDescriptorFile())) {
```
Comment (bobbrow) [inline Extension/src/LanguageServer/client.ts]: This belongs in common.ts (probably just merged in with getVcpkgRoot()).  We don't want to do this N times for N clients in a multi-root workspace.

**Review by bobbrow — CHANGES_REQUESTED** [review]

File: Extension/CHANGELOG.md (hunk 1 above)
Comment (sean-mcmanus) [reply Extension/CHANGELOG.md]: You mean "* Fix infinite loop." or something else?

Comment (bobbrow) [reply Extension/CHANGELOG.md]: yes.
"Fix infinite loop with \`#include\` code actions when no folder is open."

**Review by bobbrow — APPROVED** [review]

File: Extension/CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,12 @@
 # C/C++ for Visual Studio Code Change Log
 
+## Version 0.17.7: July 16, 2018
+* Add `${vcpkgRoot}` variable. [#1817](https://github.com/Microsoft/vscode-cpptools/issues/1817)
+* Skip automatic parsing of source files in Mac system framework paths. [#2156](https://github.com/Microsoft/vscode-cpptools/issues/2156)
+* Fix `Edit Configuration` not working after `c_cpp_properties.json` is deleted. [#2214](https://github.com/Microsoft/vscode-cpptools/issues/2214)
```
Comment (bobbrow) [inline Extension/CHANGELOG.md]: I believe the command is actually `Edit Configurations...`

Comment (sean-mcmanus) [reply Extension/CHANGELOG.md]: Fixed.

**Pull request merged**